utilities/include/vec3.hpp

```diff
@@ -1,120 +1,131 @@
+#ifndef VEC3.HPP
```

Comment:
Use underscores instead of full stops in the header guard name. In this case, we would like `VEC3_HPP` instead of `VEC3.HPP`.